specs/workspace_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'minitest'
+require_relative 'test_helper.rb'
+
+describe "Workspace" do
+  before do
+    VCR.use_cassette("load_workspace") do
+      response = SlackAPI::User.load
+      response2 = SlackAPI::Channel.load
+      @workspace = SlackAPI::Workspace.new(users: response, channels: response2)
+     end
+  end
+
+  describe "select_user" do
```

Comment:
I'd like a test on what happens if the user doesn't exist.  Also, do you lose your selection if you attempt to select an invalid user?  Should you?